frontend/packages/ux-editor/src/components/Properties/GroupConfigPanel/GroupConfigPanel.tsx

```diff
@@ -4,19 +4,23 @@ import classes from './GroupConfigPanel.module.css';
 import { useTranslation } from 'react-i18next';
 import { StudioSectionHeader } from '@studio/components-legacy';
 import { FileIcon } from '@studio/icons';
-import { useAppContext } from '../../../hooks';
+import type { ItemType } from '../ItemType';
+import type { SelectedItem } from '../../../AppContext';
 
-export const GroupConfigPanel = () => {
+type GroupConfigPanelProps = {
+  selectedItem: Extract<SelectedItem, { type: ItemType.Group }>;
+};
+
+export const GroupConfigPanel = ({ selectedItem }: GroupConfigPanelProps) => {
   const { t } = useTranslation();
-  const { selectedItem } = useAppContext();
 
   return (
     <>
       <StudioSectionHeader
         data-testid='groupConfigPanel'
         icon={<FileIcon />}
         heading={{
-          text: selectedItem?.id,
+          text: selectedItem?.id.toString(),
```

Comment:
```suggestion
          text: selectedItem.id.toString(),
```

Is it necessary with optional chaining. Shouldn't it always have a id when selected?

Reply:
True!